hasura/metadata/databases/default/tables/public_app_metadata.yaml

```diff
@@ -121,10 +121,12 @@ select_permissions:
   - role: service
     permission:
       columns:
-        - name
+        - hero_image_url
+        - id
         - logo_img_url
+        - name
+        - review_message
         - showcase_img_urls
```

Comment:
These changes aren't needed anymore since useGetVerificationDataQuery is client side now right?